### PR TITLE
[zh] Update reference\glossary (5)

### PR DESCRIPTION
**Pull request by ydFu**
Update about reference/glossary,this chapter is often used, so make adjustments,and ensure all these files were still in sync with English upstream now.

the following
```
content/zh-cn/docs/reference/glossary/cluster-operations.md
content/zh-cn/docs/reference/glossary/cluster-operator.md
content/zh-cn/docs/reference/glossary/cluster.md
content/zh-cn/docs/reference/glossary/cncf.md
```

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 1]
```diff
@@ -25,17 +24,17 @@ aka:
...
 -->
 
 <!-- 
 A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
 that run containerized applications. Every cluster has at least one worker node.
 -->
-集群是由一组被称作{{< glossary_tooltip text="节点（node）" term_id="node" >}}的机器组成，
-这些节点上会运行由 Kubernetes 所管理的容器化应用。
-且每个集群至少有一个工作节点。
+一组工作机器，称为 {{<glossary_tooltip text="节点" term_id="node" >}}，
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/cluster.md:33]:
```suggestion
一组工作机器，称为 {{< glossary_tooltip text="节点" term_id="node" >}}，
```

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 2]
```diff
@@ -44,7 +43,8 @@ nodes and the Pods in the cluster. In production environments, the control plane
 runs across multiple computers and a cluster usually runs multiple nodes, providing
 fault-tolerance and high availability.
 -->
-工作节点会托管所谓的 Pods，而 Pod 就是作为应用负载的组件。
-控制平面管理集群中的工作节点和 Pods。
-为集群提供故障转移和高可用性，
-这些控制平面一般跨多主机运行，而集群也会跨多个节点运行。
+工作程序节点托管 {{<glossary_tooltip text="Pod" term_id="pod" >}} 是
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/cluster.md:46]:
```suggestion
工作程序节点托管 {{< glossary_tooltip text="Pod" term_id="pod" >}} 是
```

File: content/zh-cn/docs/reference/glossary/cncf.md [hunk 3]
```diff
@@ -26,18 +21,25 @@ short_description: >
...
+Kubernetes 是一个云原生计算基金会项目.
 
 <!--more-->
 
 <!--
 The CNCF is a sub-foundation of [the Linux Foundation](https://www.linuxfoundation.org/).
 Its mission is to make cloud native computing ubiquitous.
 -->
-云原生计算基金会（CNCF）是 [Linux 基金会](https://www.linuxfoundation.org/) 的下属基金会。它的使命是让云原生计算无处不在。
+云原生计算基金会（CNCF）是 [Linux 基金会](https://www.linuxfoundation.org/) 的子基金会。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/cncf.md:44]:
```suggestion
云原生计算基金会（CNCF）是 [Linux 基金会](https://www.linuxfoundation.org/)的子基金会。
```

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 4]
```diff
@@ -44,7 +43,8 @@ nodes and the Pods in the cluster. In production environments, the control plane
...
+应用程序工作负载的组件。
+这{{<glossary_tooltip text="控制平面" term_id="control-plane" >}}管理集群中的工作节点和 Pod。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/cluster.md:48]:
```suggestion
这{{< glossary_tooltip text="控制平面" term_id="control-plane" >}}管理集群中的工作节点和 Pod。
```

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 5]
```diff
@@ -44,7 +42,8 @@ nodes and the Pods in the cluster. In production environments, the control plane
 runs across multiple computers and a cluster usually runs multiple nodes, providing
 fault-tolerance and high availability.
 -->
-工作节点会托管所谓的 Pods，而 Pod 就是作为应用负载的组件。
-控制平面管理集群中的工作节点和 Pods。
-为集群提供故障转移和高可用性，
-这些控制平面一般跨多主机运行，而集群也会跨多个节点运行。
+工作程序节点托管 {{< glossary_tooltip text="Pod" term_id="pod" >}} 是
+应用程序工作负载的组件。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cluster.md:46]: 这句话汉语还能读么？

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 6]
```diff
@@ -44,7 +42,8 @@ nodes and the Pods in the cluster. In production environments, the control plane
...
+这{{< glossary_tooltip text="控制平面" term_id="control-plane" >}}管理集群中的工作节点和 Pod。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cluster.md:47]: 为什么要加一个“这”？

File: content/zh-cn/docs/reference/glossary/cncf.md [hunk 7]
```diff
@@ -26,18 +21,25 @@ short_description: >
...
+
+<!--
  The Cloud Native Computing Foundation (CNCF) builds sustainable ecosystems and
  fosters a community around [projects](https://www.cncf.io/projects/) that
  orchestrate containers as part of a microservices architecture.
 
 Kubernetes is a CNCF project.
 -->
+云原生计算基金会（CNCF；Cloud Native Computing Foundation）建立了可持续的生态系统，并在围绕着[项目](https://www.cncf.io/projects/)
+建立一个社区，将容器编排为微服务架构的一部分。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cncf.md:34]:
```suggestion
建立一个社区，对作为微服务架构之组件的容器进行编排。
```

File: content/zh-cn/docs/reference/glossary/cncf.md [hunk 8]
```diff
@@ -26,18 +21,25 @@ short_description: >
...
+
+Kubernetes 是一个云原生计算基金会项目.
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cncf.md:36]:
```suggestion
Kubernetes 是一个 CNCF 项目.
```

File: content/zh-cn/docs/reference/glossary/cluster.md [hunk 9]
```diff
@@ -25,17 +23,17 @@ aka:
...
 -->
 
 <!-- 
 A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
 that run containerized applications. Every cluster has at least one worker node.
 -->
-集群是由一组被称作{{< glossary_tooltip text="节点（node）" term_id="node" >}}的机器组成，
-这些节点上会运行由 Kubernetes 所管理的容器化应用。
-且每个集群至少有一个工作节点。
+一组工作机器，称为 {{< glossary_tooltip text="节点" term_id="node" >}}，
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/cluster.md:32]:
```suggestion
一组工作机器，称为{{< glossary_tooltip text="节点" term_id="node" >}}，
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/docs/reference/glossary/cncf.md [hunk 10]
```diff
@@ -26,18 +21,25 @@ short_description: >
...
+
+<!--
  The Cloud Native Computing Foundation (CNCF) builds sustainable ecosystems and
  fosters a community around [projects](https://www.cncf.io/projects/) that
  orchestrate containers as part of a microservices architecture.
 
 Kubernetes is a CNCF project.
 -->
+云原生计算基金会（CNCF；Cloud Native Computing Foundation）建立了可持续的生态系统，并在围绕着[项目](https://www.cncf.io/projects/)
+建立一个社区，对作为微服务架构之组件的容器进行编排。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cncf.md:34]:
```suggestion
云原生计算基金会（CNCF；Cloud Native Computing Foundation）建立可持续的生态系统，
并在围绕[项目](https://www.cncf.io/projects/)建立社区，对作为微服务架构之组件的容器进行编排。
```

File: content/zh-cn/docs/reference/glossary/cluster.md (hunk 9 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/glossary/cluster.md:32]: This is a reasonable suggestion, why not fixed?

File: content/zh-cn/docs/reference/glossary/cncf.md [hunk 11]
```diff
@@ -26,18 +21,25 @@ short_description: >
...
+
+<!--
  The Cloud Native Computing Foundation (CNCF) builds sustainable ecosystems and
  fosters a community around [projects](https://www.cncf.io/projects/) that
  orchestrate containers as part of a microservices architecture.
 
 Kubernetes is a CNCF project.
 -->
+云原生计算基金会（CNCF；Cloud Native Computing Foundation）建立可持续的生态系统，
+并在围绕[项目](https://www.cncf.io/projects/)建立社区，对作为微服务架构之组件的容器进行编排。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cncf.md:34]:
```suggestion
并围绕[项目](https://www.cncf.io/projects/)建立社区，对作为微服务架构之组件的容器进行编排。
```

**Pull request merged**